packages/netlify-cms-app/package.json

```diff
@@ -31,6 +31,7 @@
     "immutable": "^3.7.6",
     "lodash": "^4.17.11",
     "moment": "^2.24.0",
+    "netlify-cms-backend-azure": "^2.5.0",
```

Comment:
Any reason for not starting the package from version `1.0.0`?

Reply:
Reset to 1.0.0 in latest commit